package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/lambda-tools",
-  "version": "6.0.1",
+  "version": "7.0.0-beta",
```

Comment:
This is a placeholder so make sure that no 6.x versions are released with an 5.5 MockServer library in them